.github/workflows/admin-pr.yml

```diff
@@ -23,13 +20,13 @@ jobs:
           firebase setup:emulators:ui
 
       - name: Install Dependencies
-        run: npm install
+        run: npm --workspace=@socialincome/admin install
 
       - name: Run Tests
-        run: npm run emulators:test
+        run: npm run admin:test
 
       - name: Build
-        run: npm run build
+        run: npm run admin:build
```

Comment:
@mkue: I think overriding the env vars doesn't work any more. That's the reason the preview (and then the final deployment, I assume) is broken.

Might be related to the dotenv config?
```
		new Dotenv({
 			path: path.resolve(path.join(__dirname, '.env')),
 		}),
```

We can also change the way how we set the production variable. The current setup is not nice:
a) we have code duplication in admin-pr and admin-deployment github workflow
b) we nullify the auth envs here

In the backend deployment, I write the github secrets to a .env file. We could do the same here, then your dotenv config should work again.

We could remove the `admin-deployment` workflow and use the `admin-pr` and in the deployment step have an if branch==main => channelId: live.

Wdyt?

Reply:
@mkue, I added a few commits for the github workflows. Hope that's fine. The preview works now again, and we only have one yaml for PR and deployment.

Reply:
super nice, thanks!